src/pages/settings/panes/Audio.tsx

```diff
@@ -40,21 +40,32 @@ export function Component() {
         try {
             const { state } = await navigator.permissions.query({
                 // eslint-disable-next-line
-                // @ts-ignore
+                // @ts-ignore: very few browsers accept this `PermissionName`, but it has been drafted in https://www.w3.org/TR/permissions/#powerful-features-registry
                 name: "microphone",
             });
 
             setPermission(state);
         } catch (err) {
-            // the browser might not support `query` functionnality
-            setError(err as DOMException);
+            // the browser might not support `query` functionnality or `PermissionName`
+            // nothing to do
         }
     };
 
     useEffect(() => {
         askOrGetPermission();
     }, []);
 
+    useEffect(() => {
+        return () => {
```

Comment:
alternatively we could close the access right after getting the list of microphone.

And I'm still annoyed by the fact the user can change page before accept/refuse microphone access and fail to apply the state correctly.

If you have better idea, tell me.

Reply:
I think I will keep it as-is right now and fix any major issues as they arise later.